pom.xml

```diff
@@ -194,7 +195,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
-                    <argLine>-XX:-UseSplitVerifier</argLine>
+                    <argLine>${argLine} -XX:-UseSplitVerifier</argLine>
```

Comment:
Because the command line argument -UseSplitVerifier was deprecated with Java 8. The connector is supposed to be runable with Java 7, so I decided to keep it in this project. That being said though, I'm not sure why we added the flag in the first place. Might it have something to do with the usage of Corbertura? If so we can remove it here too, since we switched to Jacoco.